src/cclib/io/filewriter.py

```diff
@@ -89,6 +92,13 @@ def _make_bond_connectivity_from_openbabel(self, obmol):
                                         obbond.GetBondOrder()))
         return bond_connectivities
 
+    def _check_required_attributes(self, required):
```

Comment:
Addressed. `required_attrs` is now created as a class attribute in the [subclass](https://github.com/cclib/cclib/pull/391/files#diff-599bcb2b83c5ef046682dfaa37fd74e1R33).